spec/phlex/component_spec.rb

```diff
@@ -123,6 +123,26 @@ def template
     end
   end
 
+  describe "with unsafe attributes" do
+    let :component do
+      Class.new Phlex::Component do
+        def template
+          article **attributes
```

Comment:
It’s not likely this would ever happen, but the point is it could and we should check.